lte/gateway/deploy/roles/magma/tasks/main.yml

```diff
@@ -231,6 +231,14 @@
       # install openvswitch
       - openvswitch-switch
       - openvswitch-test
+      - libopenvswitch
```

Comment:
It would be good to verify this with building local package, to confirm there's no dependency issue?